gradoop-flink/src/main/java/org/gradoop/flink/datagen/transactions/foodbroker/generators/EmployeeGenerator.java

```diff
@@ -45,9 +46,13 @@ public EmployeeGenerator(
 
   @Override
   public DataSet<Vertex> generate() {
-    List<MasterDataSeed> seeds = getMasterDataSeeds(Employee.CLASS_NAME);
+    List<MasterDataSeed> seeds = getMasterDataSeeds(Constants.EMPLOYEE_VERTEX_LABEL);
     List<String> cities = foodBrokerConfig
       .getStringValuesFromFile("cities");
+    List<String> companies = foodBrokerConfig
```

Comment:
Are companies and holdings required for Employee generation? 
They all belong to the FoodBroker company...